### DISCUSSION TITLE
Setting up SendPulse with next-auth email provider

### DISCUSSION THREAD

**Discussion by v1rtl** [nextauthjs/next-auth#1844] [Help]
Hello,
I'm trying to connect SendPulse with next-auth with this code:

```js
    Providers.Email({
      
      server: {
        host: 'smtp-pulse.com',
        port: 465,
        auth: {
          user: process.env.SENDPULSE_USER,
          pass: process.env.SENDPULSE_PWD
        }
      },
      from: process.env.SENDPULSE_USER
    })
```

and I get this:

```js
{
  code: 'EENVELOPE',
  response: '554 5.9.2 Sender domain is not valid.',
  responseCode: 554,
  command: 'MAIL FROM'
}
```

both password and email address are valid and I cannot understand why the error appears. What's the proper way of making it work?

**Comment by ibraheem88** [comment]
In from section make sure that email entered is added to SenPulse sender emails. I had a different email in from and it was causing this error.
